src/polkitbackend/polkitbackendauthority.c

```diff
@@ -1591,3 +1598,12 @@ polkit_backend_authority_log (PolkitBackendAuthority *authority,
 
   g_free (message);
 }
+
+void
+polkit_backend_authority_set_log_level (const gint level)
+  {
+    if ((level >= 0) && (level <= LOG_LEVEL_VERBOSE))
```

Comment:
Boolean operators have lower precedence than comparison operators, so those nested parens are not needed.

Reply:
Yes, you're right, but it's clearer to the reader and immediately visible. That's one of the "will do more than I have to just to make it prettier for humans" LifeProTips on my list :D